cmd/libsnap-confine-private/classic-test.c

```diff
@@ -20,4 +20,33 @@
 
 #include <glib.h>
 
-// TODO: write some tests
+const char *os_release_classic = ""
+    "NAME=\"Ubuntu\"\n"
+    "VERSION=\"17.04 (Zesty Zapus)\"\n" "ID=ubuntu\n" "ID_LIKE=debian\n";
+
+static void test_is_on_classic()
+{
+	g_file_set_contents("os-release.classic", os_release_classic,
+			    strlen(os_release_classic), NULL);
+	os_release = "os-release.classic";
+	g_assert_true(is_running_on_classic_distribution());
+	unlink("os-release.classic");
+}
+
+const char *os_release_core = ""
+    "NAME=\"Ubuntu Core\"\n" "VERSION=\"16\"\n" "ID=ubuntu-core\n";
+
+static void test_is_on_core()
```

Comment:
Can you add a test that shows what happens when the line is longer than 255 bytes please?